userspace/falco/configuration.cpp

```diff
@@ -72,7 +72,7 @@ falco_configuration::falco_configuration():
 	m_metrics_interval(5000),
 	m_metrics_stats_rule_enabled(false),
 	m_metrics_output_file(""),
-	m_metrics_flags((METRICS_V2_KERNEL_COUNTERS | METRICS_V2_LIBBPF_STATS | METRICS_V2_RESOURCE_UTILIZATION | METRICS_V2_STATE_COUNTERS | METRICS_V2_RULE_COUNTERS)),
+	m_metrics_flags((METRICS_V2_KERNEL_COUNTERS | METRICS_V2_LIBBPF_STATS | METRICS_V2_RESOURCE_UTILIZATION | METRICS_V2_STATE_COUNTERS | METRICS_V2_RULE_COUNTERS | METRICS_V2_PLUGINS)),
```

Comment:
Question: why are we initialing `m_metrics_flags` with everything enabled, considering that we are going to reset it to `0` before loading the really enabled ones? https://github.com/falcosecurity/falco/pull/3228/files#diff-9bd683a06f316f8dab7df7c882f9d1857df11de6ae476356e5f2f86068891157R537

Wouldn't it be better to initialize it to `0`?
@incertum

Reply:
Good point, yes it does not make much sense.